src/Aspire.Hosting/Dcp/DcpVersion.cs

```diff
@@ -0,0 +1,10 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Aspire.Hosting.Dcp;
+
+public static class DcpVersion
+{
+    public static Version MinimumVersionInclusive = new Version(0, 1, 43);
```

Comment:
```suggestion
    public static Version MinimumVersionInclusive = new Version(0, 1, 44);
```

Reply:
0.1.44 is the first one with a version string that doesn't include a `+5`, which does not parse.